Big-Spender/readme.md

```diff
@@ -124,8 +124,39 @@ CREATE YOUR QUERY HERE
 **You:** Sure thing. To confirm, the date is August 19, 2021, the transaction number is 38104091, the supplier invoice number is 3780119655, the supplier is 'Dell', the expense type is 'Hardware' and the expense area is 'IT'. Here's the query for that:
 
 ```sql
-INSERT YOUR QUERIES HERE
-
+-- Ensure the expense type exists
+INSERT INTO expense_types (expense_type)
+SELECT 'Hardware'
+WHERE NOT EXISTS (
+    SELECT 1 FROM expense_types WHERE expense_type = 'Hardware'
```

Comment:
Great attention to detail! The 'Hardware', 'Dell', and 'IT' rows were missing from the other tables, and you've done well by adding the necessary insert statements to populate them.